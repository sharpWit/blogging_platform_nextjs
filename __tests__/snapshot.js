import { render } from "@testing-library/react";
import HomePage from "../src/app/(home)/page.tsx";

it("renders homepage unchanged", () => {
  const { container } = render(<HomePage />);
  expect(container).toMatchSnapshot();
});
