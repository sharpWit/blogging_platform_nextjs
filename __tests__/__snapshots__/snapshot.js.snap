// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="mx-auto w-full max-w-screen-xl px-2.5 md:px-20"
  >
    <main
      class="flex flex-col items-start justify-evenly mt-16 md:flex-row"
    >
      <div>
        <h1>
          Home
        </h1>
        <p>
          Contents
        </p>
      </div>
      <div
        class="h-screen"
      >
        <div>
          <h1
            class="font-bold mb-4"
          >
            TOP CATEGORIES
          </h1>
          <div
            class="grid grid-cols-[repeat(auto-fill,minmax(100px,1fr))] gap-2"
          >
            <a
              class="inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium ring-offset-background focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 bg-secondary text-secondary-foreground hover:bg-secondary/80 h-10 px-4 py-2 hover:scale-110 transition-all"
              href="/posts"
            >
              Posts
            </a>
          </div>
          <div
            class="min-h-screen p-8 pb-20 sm:p-20 font-[family-name:var(--font-geist-sans)]"
          />
        </div>
      </div>
    </main>
  </div>
</div>
`;
